modules/library/src/main/scala/zio/elasticsearch/query/Queries.scala

```diff
@@ -782,6 +781,234 @@ private[elasticsearch] final case class Ids[S](values: Chunk[String]) extends Id
     Obj("ids" -> Obj("values" -> Arr(values.map(_.toJson))))
 }
 
+sealed trait IntervalQuery {
+  private[elasticsearch] def toJson: Json
+}
```

Comment:
This should probably be renamed to `IntervalRule`?
And this, and all classes extending it should be moved out from Queries file.